lib/utilities/pristine.js

```diff
@@ -246,6 +247,23 @@ function addEmberDataToDependencies(appName, version) {
   };
 }
 
+function removeEmberFromDependencies(appName, version) {
```

Comment:
if we go down this path can we rename this to `removeEmberSourceFromDependencies`?